Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rust:1.67
```

Comment:
There seems to be a lot of discussions and arguments both for and against using `:latest` in general  and the reason `:1.67` was used was mainly because the official `rust` image used it in an example. However, i agree that it would be better to use `:latest` in our case.